src/anyp/PortCfg.cc

```diff
@@ -101,13 +131,19 @@ AnyP::PortCfg::codeContextGist() const
 
 std::ostream &
 AnyP::PortCfg::detailCodeContext(std::ostream &os) const
+{
+    os << Debug::Extra << "listening port: " << *this;
+    return os;
+}
+
+void
+AnyP::PortCfg::print(std::ostream &os) const
 {
     // parsePortSpecification() defaults optional port name to the required
```

Comment:
If all callers benefit from it, let's add printing the newly added directive name to this method.